src/effects/Effect.cpp

```diff
@@ -68,11 +68,6 @@ Effect::Effect()
 
 Effect::~Effect()
 {
-   // Destroying what is usually the unique Effect object of its subclass,
-   // which lasts until the end of the session.
-   // Maybe there is a non-modal realtime dialog still open.
-   if (mHostUIDialog)
```

Comment:
I wonder if should revisit the modality of destructive dialogs though. It appears that we have broken a common workflow by making destructive application dialogs modal. 

This won't really matter when automation is introduced, so probably it doesn't really matters.